### PR TITLE
finish we did it dumb

### PR DESCRIPTION
**Pull request by RahulTirumalareddy**

### CODE REVIEW

File: _posts/2019-04-13-lookin-playboi-carti.md [hunk 1]
```diff
@@ -33,9 +33,10 @@ allow-storage-access-by-user-activation allow-top-navigation-by-user-activation"
 src="https://embed.music.apple.com/us/album/lookin-feat-lil-uzi-vert/1224307213?i=1224307239&app=music">
 </iframe>
 
-The sonic equivalent to the first picture of a black hole, *Lookin* wastes no
-time in discussing the crux of the matter with its opening line (don’t worry if
-you missed it the first time; it comprises perhaps half the song)
+The sonic equivalent to the first picture of a black hole,
+[*Lookin*](https://youtu.be/lQd-oFbTylc) wastes no time in discussing the crux
```
Comment (mingyli) [inline _posts/2019-04-13-lookin-playboi-carti.md]: remove 'in'

File: _posts/2019-04-13-lookin-playboi-carti.md [hunk 2]
```diff
@@ -33,9 +33,10 @@ allow-storage-access-by-user-activation allow-top-navigation-by-user-activation"
...
+of the matter with its opening line (don’t worry if you missed it the first
+time; it comprises perhaps half the song)
 
 > Look at these n[\*\*]s.
```
Comment (mingyli) [inline _posts/2019-04-13-lookin-playboi-carti.md]: Mayhaps this should be in a separate pr

File: _posts/2019-04-13-lookin-playboi-carti.md [hunk 3]
```diff
@@ -33,9 +33,10 @@ allow-storage-access-by-user-activation allow-top-navigation-by-user-activation"
 src="https://embed.music.apple.com/us/album/lookin-feat-lil-uzi-vert/1224307213?i=1224307239&app=music">
 </iframe>
 
-The sonic equivalent to the first picture of a black hole, *Lookin* wastes no
-time in discussing the crux of the matter with its opening line (don’t worry if
-you missed it the first time; it comprises perhaps half the song)
+The sonic equivalent to the first picture of a black hole,
```
Comment (mingyli) [inline _posts/2019-04-13-lookin-playboi-carti.md]: What does this even mean lol

File: _posts/2019-04-17-owengamingboy.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+---
+layout:     post
+title:      "Deconstructivist Pop: A Case Study"
+author:     Rahul Tirumalareddy
+date:       2019-04-17
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: Shouldn't this be May 25?

File: _posts/2019-04-17-owengamingboy.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+categories: Owengamingboy
+album_art:  assets/images/we_did_it_dumb.jpg
+---
+
+Macaroni and cheese. Peanut butter and jelly. Hide and seek. Cookies and milk.
+Research and development. Crime and Punishment. Surf and sand. Fish and chips.
+Cheeseburgers and fries. Mom and dad. Marriage and divorce. Weekdays and
+weekends. The soccer field and Ben & Jerry's. Mom and another guy. Milk and
+honey. Hiroshima and Nagasaki. Some things go together like peas in a pod.
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: LOL how long did this take?

File: _posts/2019-04-17-owengamingboy.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+Research and development. Crime and Punishment. Surf and sand. Fish and chips.
+Cheeseburgers and fries. Mom and dad. Marriage and divorce. Weekdays and
+weekends. The soccer field and Ben & Jerry's. Mom and another guy. Milk and
+honey. Hiroshima and Nagasaki. Some things go together like peas in a pod. 
+
+But when mentioning things that just go together, how could one fail to mention
+[Owengamingboy](https://www.youtube.com/channel/UCCxPYICbpS1M-WWv9RgVQNg/) and
+[YouTube](https://www.youtube.com)? Who is Owengamingboy, you may ask (if you've
+been living under a rock for the last few years)? Owengamingboy is the online
+moniker for a modern Renaissance man, or rather, boy skilled in Fortnite, music,
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: replace 'or rather' with something else, like a parenthetical

File: _posts/2019-04-17-owengamingboy.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+genius has not escaped the public eye, and already has 19 views. Any more, I
+fear, and YouTube's view counter may break.  
+
+<iframe width="85%" height="315" src="https://www.youtube.com/embed/qVGvEgnormY"
+frameborder="0" controls="0" allowfullscreen></iframe>
+
+From the get-go, Owengamingboy wastes no time in subverting traditional music
+video expectations, as he leads with an all-black intro screen, save for the
+words "Untitled" stitched across the screen. In this era of commercialization,
+logos, and branding, Owengamingboy throws all expectations for revenue
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: SUBVERTS EXPECTATIONS

File: _posts/2019-04-17-owengamingboy.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+Moving onto the song itself, Owengamingboy starts off with the question,
+
+> Yo! What's two plus two times half?
+
+Perhaps one of the best opening verses in the last century, this inquiry may
+seem easy enough to answer, but is, in fact, undecidable, as the half could be
+applied to the whole sum, or just the latter two. Both readings are equally
+valid, and throughout the song Owengamingboy forces the audience to come to
+terms with uncertainty as he does here.
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: Order of operations 8)

File: _posts/2019-04-17-owengamingboy.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+Macaroni and cheese. Peanut butter and jelly. Hide and seek. Cookies and milk.
+Research and development. Crime and Punishment. Surf and sand. Fish and chips.
+Cheeseburgers and fries. Mom and dad. Marriage and divorce. Weekdays and
+weekends. The soccer field and Ben & Jerry's. Mom and another guy. Milk and
+honey. Hiroshima and Nagasaki. Some things go together like peas in a pod. 
+
+But when mentioning things that just go together, how could one fail to mention
+[Owengamingboy](https://www.youtube.com/channel/UCCxPYICbpS1M-WWv9RgVQNg/) and
+[YouTube](https://www.youtube.com)? Who is Owengamingboy, you may ask (if you've
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: I don't like the YouTube hyperlink

File: _posts/2019-04-17-owengamingboy.md (hunk 6 above)
Comment (mingyli) [reply _posts/2019-04-17-owengamingboy.md]: At the very least, needs a comma after 'boy'

File: _posts/2019-04-17-owengamingboy.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+been living under a rock for the last few years)? Owengamingboy is the online
+moniker for a modern Renaissance man, or rather, boy skilled in Fortnite, music,
+and Fortnite. Breaking onto the YouTube scene on July 23, 2016, the young man
+has had a meteoric upswing in popularity in a short two years, boasting over 90
+subscribers at the time of writing. A brief perusal of his portfolio reveals a
+myriad of videos called *Fortnite 100 Subsriber suprise countdown*.
```
Comment (mingyli) [inline _posts/2019-04-17-owengamingboy.md]: I only see one such video with that title.

**Comment by mingyli** [comment]
This is cyberbullying

File: _posts/2019-04-17-owengamingboy.md (hunk 5 above)
Comment (mingyli) [reply _posts/2019-04-17-owengamingboy.md]: Remove sensitive topic

File: _posts/2019-04-13-lookin-playboi-carti.md (hunk 3 above)
Comment (RahulTirumalareddy) [reply _posts/2019-04-13-lookin-playboi-carti.md]: he is making music the likes of which have never been heard before

File: _posts/2019-04-17-owengamingboy.md (hunk 4 above)
Comment (RahulTirumalareddy) [reply _posts/2019-04-17-owengamingboy.md]: I started writing the article at that date

File: _posts/2019-04-17-owengamingboy.md (hunk 9 above)
Comment (RahulTirumalareddy) [reply _posts/2019-04-17-owengamingboy.md]: removed

File: _posts/2019-04-17-owengamingboy.md (hunk 10 above)
Comment (RahulTirumalareddy) [reply _posts/2019-04-17-owengamingboy.md]: there are many
Screen Shot 2019-05-26 at 4 47 05 PM

Comment (mingyli) [reply _posts/2019-04-17-owengamingboy.md]: ah my bad I must have misread

File: _posts/2019-04-13-lookin-playboi-carti.md (hunk 2 above)
Comment (RahulTirumalareddy) [reply _posts/2019-04-13-lookin-playboi-carti.md]: i am certainly lazy

File: _posts/2019-04-17-owengamingboy.md (hunk 4 above)
Comment (RahulTirumalareddy) [reply _posts/2019-04-17-owengamingboy.md]: actually yea, it makes more sense to have the finish date, will update

File: _posts/2019-04-13-lookin-playboi-carti.md (hunk 2 above)
Comment (mingyli) [reply _posts/2019-04-13-lookin-playboi-carti.md]: 8)

File: _posts/2019-05-26-owengamingboy.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+date:       2019-05-26
+categories: Owengamingboy
+album_art:  assets/images/we_did_it_dumb.jpg
+---
+
+Macaroni and cheese. Peanut butter and jelly. Hide and seek. Cookies and milk.
+Research and development. Crime and Punishment. Surf and sand. Fish and chips.
+Cheeseburgers and fries. Mom and dad. Marriage and divorce. Weekdays and
+weekends. The soccer field and Ben & Jerry's. Mom and another guy. Milk and
+honey. Hiroshima and Nagasaki. Some things go together like peas in a pod.
```
Comment (mingyli) [inline _posts/2019-05-26-owengamingboy.md:14]: You don't want your recruiter to see this one

Comment (RahulTirumalareddy) [reply _posts/2019-05-26-owengamingboy.md:14]: yes i do.

Comment (RahulTirumalareddy) [reply _posts/2019-05-26-owengamingboy.md:14]: it displays my wit

**Review by mingyli — APPROVED** [review]
I approve but let it be known that I disagree with your final polar example
